src/providers/Auth/Auth.provider.jsx

```diff
@@ -14,27 +14,40 @@ function useAuth() {
 }
 
 function AuthProvider({ children }) {
-  const [authenticated, setAuthenticated] = useState(false);
+  const [admin, setAdmin] = useState(false);
+  const [employee, setEmployee] = useState(false);
 
   useEffect(() => {
-    const lastAuthState = storage.get(AUTH_STORAGE_KEY);
-    const isAuthenticated = Boolean(lastAuthState);
+    const lastAdminState = storage.get(ADMIN_STORAGE_KEY);
+    const lastEmployeeState = storage.get(EMPLOYEE_STORAGE_KEY);
+    const isAdmin = Boolean(lastAdminState);
+    const isEmployee = Boolean(lastEmployeeState);
 
-    setAuthenticated(isAuthenticated);
+    setAdmin(isAdmin);
+    setEmployee(isEmployee);
   }, []);
 
   const login = useCallback(() => {
-    setAuthenticated(true);
-    storage.set(AUTH_STORAGE_KEY, true);
+    if (1 + Math.floor(2 * Math.random()) === 1) {
```

Comment:
I think that we should fix this

Reply:
There is an opened ticket that is related to this. (Remove hardcoded) #14

Reply:
So, I will merge it, and until we have the backend ready I will fix it!